repos.txt

```diff
@@ -111,6 +117,7 @@ java-tasks
 java-texttospeech
 java-tpu
 java-trace
+java-translate
```

Comment:
It's GAPIC auto-generated library. It copies code from googleapis-gen (configuration: https://github.com/googleapis/java-translate/blob/main/.github/.OwlBot.yaml)